src/main/java/Panna.java

```diff
@@ -0,0 +1,161 @@
+            }
+
+
+            else if (Panna.command.equals("mark")) {
+                System.out.println("Which one should I mark? Write the label number :] ");
+                try {
+                    int label = s.nextInt();
+                    tasks.get(label - 1).setDone(true);
+
+                    System.out.println("----------------------------------------------------------");
+                    System.out.println("Nice! I've marked this task as done: \n"
+                            + tasks.get(label - 1));
+                    System.out.println("----------------------------------------------------------");
+
+                }
+                catch (Exception e) {
+                    throw new PannaException("Invalid label! The number of tasks now is " + tasks.size() +
+                            "\nPlease try with a more appropriate value! ");
+                }
+            }
+
+            else if (Panna.command.equals("unmark")) {
+                System.out.println("Which one should I unmark? Write the label number :] ");
+                try {
+                    int label = s.nextInt();
+                    tasks.get(label - 1).setDone(false);
+
+                    System.out.println("----------------------------------------------------------");
+                    System.out.println("Okay! I've unmarked this task as done: \n"
+                            + tasks.get(label - 1));
+                    System.out.println("----------------------------------------------------------");
+                }
+                catch (Exception e) {
+                    throw new PannaException("Invalid label! The number of tasks now is" + tasks.size() +
+                                                "\nPlease try with a more appropriate value! ");
+                }
+
+            }
+
+            else if (Panna.command.equals("event")) {
+                try {
+                    System.out.println("What kind of event? ");
+                    String input = s.nextLine();
+                    System.out.println("When does it start? ");
+                    String start = s.nextLine();
+                    System.out.println("When does it end? ");
+                    String end = s.nextLine();
+                    Task t = new Event(input, start, end);
+                    tasks.add(t);
+                    System.out.println("Got it! I've added the \n" + t + "\n event!");
+                    System.out.println("Now you have " + tasks.size() + " task(s) in the list! ");
+                }
+                catch (Exception e) {
+                    throw new PannaException("All inputs must be Strings! Please ensure it is not empty :D");
+                }
+
+            }
+
+            else if (Panna.command.equals("todo")) {
+                try {
+                    System.out.println("What kind of todo? ");
+                    String input = s.nextLine();
+                    Task t = new Todo(input);
```

Comment:
Perhaps naming the variable more intuitively would help for readability!
```suggestion
                    Task task = new Todo(input);
```